sharding-jdbc/sharding-jdbc-core/src/main/java/org/apache/shardingsphere/shardingjdbc/jdbc/core/constant/SQLExceptionConstant.java

```diff
@@ -27,4 +27,8 @@
 public final class SQLExceptionConstant {
     
     public static final String SQL_STRING_NULL_OR_EMPTY = "SQL String can not be NULL or empty.";
+    
+    public static final String SQL_STATE = "S1002";
```

Comment:
`SQL_STATE` is a common name, but the value is `S1002`, it is not a good variable name.

Reply:
ok